test/functional/helpers.lua

```diff
@@ -959,7 +959,8 @@ end
 
 module = global_helpers.tbl_extend('error', module, global_helpers)
 
-return function(after_each)
+return function()
+  local after_each = getfenv(2).after_each
```

Comment:
Looks like we can't avoid needing to run this function for every test file. So avoiding the argument doesn't really make things easier, and has the disadvantage of being less explicit.

Reply:
I don't understand your comment. Do you want an event handler or not?

Reply:
OK, then it sounds like there's no way to move forward with this PR then.